app/main.py

```diff
@@ -1,34 +1,77 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple,
+            end: tuple,
+            is_drowned: bool = False
+    ) -> None:
+        self.start = start
+        self.end = end
+        self.is_drowned = is_drowned
+        self.decks = []
+        if start[0] == end[0] and start[1] != end[1]:
+            index = 0
+            for _ in range(start[1], end[1] + 1):
+                self.decks.append(Deck(start[0],
+                                       start[1] + index,
+                                       is_alive=True))
+                index += 1
+        elif start[1] == end[1] and start[0] != end[0]:
+            index = 0
```

Comment:
Index variable is not needed. Result will be the same if you instead _ will use index